### ISSUE TITLE
Unable to Run MQTT Example Successfully

### ISSUE THREAD

**Issue by halkony** [aklivity/zilla#724]
**Describe the bug**
After following [these instructions](https://docs.aklivity.io/zilla/latest/tutorials/mqtt/mqtt-intro.html#setup) and starting the given stack, I am unable to connect to the Kafka broker using MQTT.

I seem to be able to talk to the zilla instance, but attempting to connect to mqtt://host.docker.internal:7183 results in repeated `Client null sending CONNECT` errors

**To Reproduce**
Follow [these instructions](https://docs.aklivity.io/zilla/latest/tutorials/mqtt/mqtt-intro.html#setup) on Windows 10.

**Expected behavior**
I expected to be able to complete the tutorial as described. 
**Screenshots**
After successfully running the stack in the tutorial:
![image](https://github.com/aklivity/zilla/assets/15790371/1b24867f-c6e4-49c3-a2a7-fef08837df7f)

**Desktop (please complete the following information):**
Windows 10
Docker v4.26.1

**Additional context**
I've tried connecting with multiple MQTT clients with no luck.

Only other thing I can think to try is running Docker in Linux.

**Comment by halkony** [comment]
I am unable to run the example on Ubuntu as well. Tried version 0.9.64 and 63. 
![image](https://github.com/aklivity/zilla/assets/15790371/60682fca-ef09-438b-9f8c-2a41c0f75a1d)

**Comment by halkony** [comment]
I narrowed this down to rootless docker networking. The example is working on Ubuntu 22.04.

**Comment by vordimous** [comment]
@halkony Thank you for reporting and triaging this issue. 

> I narrowed this down to rootless docker networking. The example is working on Ubuntu 22.04.

Do you mean you were running the docker example in [rootless mode](https://docs.docker.com/engine/security/rootless/) which was causing the host network not to work as the example expects? 

"--net=host doesn't listen ports on the host network namespace" from the docker docs looks to mean the examples as written won't work in rootless mode.

**Comment by halkony** [comment]
Correct. I did get it running in rootless docker, however.

It turns out Ubuntu has permissions problems when installing docker with snap. Installing from the official docker docs via .deb fixed a lot of my issues. Additionally, I set `~/.config/docker/daemon.json` to
```
{
    "default-address-pools": [
        {"base":"172.16.0.0/16","size":24},
        {"base":"172.20.0.0/16","size":24}
    ]
}
```
so the rootless address space doesn't conflict with my localhost (this fixes the rootless 2181 zookeeper collisions I had out of the box). I also removed the
```
networks:
  default:
    driver: bridge
```
section I had added manually because I mistook bridge mode for the default host mode.

None of these seemed to explicitly solve my problem, but as is typical of linux voodoo, solving several small problems usually fixes one big one. Cheers.
